src/applications/facility-locator/components/LocationHours.jsx

```diff
@@ -41,14 +41,16 @@ const LocationHours = ({ location }) => {
 
   const facilityType = get(location, 'attributes.facilityType');
```

Comment:
doesn't location sometimes not show up? Or maybe I'm thinking of some other object from FL. Actually, because of the ternary of 45, it's fine because it'd default to Hours of operation